elliottlib/bzutil.py

```diff
@@ -60,6 +60,9 @@ def whiteboard_component(self):
     def is_tracker_bug(self):
         return set(constants.TRACKER_BUG_KEYWORDS).issubset(set(self.keywords))
 
+    def is_cve_in_summary(self):
+        return True if re.search(r'^CVE-\d+-\d+', self.summary) else False
```

Comment:
```suggestion
        return re.search(r'^CVE-\d+-\d+', self.summary)
```

Reply:
`re.search` returns a re object, joep suggests we should make the return value a clear boolean

Reply:
Ah that's true then `return bool(re.search(r'^CVE-\d+-\d+', self.summary))` should do the job right?

Reply:
create a new pr https://github.com/openshift/elliott/pull/454